docs/docs/configuration.md

```diff
@@ -180,6 +180,7 @@ Required and optional properties to include while using `google` authentication
 | Property                   | Default                                          | Description                                      |
 |----------------------------|--------------------------------------------------|--------------------------------------------------|
 | `gcp.auth.credentials-path`| Application Default Credentials (ADC)            | Path to a service account JSON key file.         |
+| `gcp.auth.credentials-key` | Application Default Credentials (ADC)            | JSON string of a service account credential.     |
```

Comment:
```suggestion
| `gcp.auth.credentials-json` | Application Default Credentials (ADC)            | JSON string of a service account credential.     |
```
Just a suggestion: There's two other GCS connectors by Confluent and Aiven that use `gcs.credentials.path` and `gcs.credentials.json` which are slightly more descriptive at first glance.  `key` by itself can be a bit confusing for the intention.

It's too bad we didn't all standardize on the same names, but that ship has sailed 😄